integr8ly/integreatly-operator
# MGDAPI-3855 - Implement persistent volume for alert manager

# Issue link
Implement persistent volume for alert manager
https://issues.redhat.com/browse/MGDAPI-3855
This PR could be merged as Observability PR (that part of this Jira): https://github.com/redhat-developer/observability-operator/pull/114 is already merged and New OO release available: v3.0.11.

# What
- Implement persistent volume for alert manager
- This PR includes change of OO version: from v3.0.8 to v3.0.11.

# Verification steps
1. Check that Alertmanager PVC created in redhat-rhoam-observability namespace. For example:
$ oc get pvc -n redhat-rhoam-observability
NAME STATUS VOLUME CAPACITY ACCESS MODES STORAGECLASS AGE
alertmanager-alertmanager-db-alertmanager-alertmanager-0 Bound pvc-60903afd-b8b0-43c1-8fc4-1adc4062fe01 1Mi RWO gp2 3h2m
prometheus-prometheus-db-prometheus-prometheus-0 Bound pvc-4ae289f2-5fa8-42f1-9864-19aa805e0760 50Gi RWO gp2 3h2m

2. Enter Alertmanager UI, Silent/Snooze any alert (for example alertname="DeadMansSwitch"), kill alertmanager pod , wait for recreation of the pod, enter again Alertmanager UI, and see that alert still snoozed

3. Notes: - The PR was verified on local installation and install from bundle  - https://quay.io/repository/vmogilev_rhmi/managed-api-service-index?tab=tags&tag=1.23.0
